src/goto-programs/goto_convert.cpp

```diff
@@ -301,6 +307,9 @@ void goto_convertt::goto_convert_rec(
   const codet &code,
   goto_programt &dest)
 {
+  // Check that guarded_gotos was cleared after any previous use of this
+  // converter instance:
+  PRECONDITION(guarded_gotos.size() == 0);
```

Comment:
Please use `.empty()` instead of comparing the size.

Reply:
Fixed, ready for re-review